config/routes.rb

```diff
@@ -21,7 +21,9 @@
   namespace :api, defaults: { format: :json } do
     namespace :v1 do
       namespace :admin do
-        resources :playlist_activities, only: [:update]
+        resources :students, only: [:update] do
```

Comment:
I dont think you need the update here, you are just using this resources hook to pull in the student id. unless you have an ajax route that updates the student record diretly instead of a nested action for just the playlist activities, i think this should be `only: []`